topics/galaxy-interface/tutorials/intermine/tutorial.md

```diff
@@ -0,0 +1,144 @@
+{: .hands_on}
+
+
+This will redirect you to the InterMine registry, which shows a full list of InterMines and the various organisms they support. Find an InterMine that has the organism type you’re working with, and click on it to redirect to that InterMine.
+
+Once you arrive at your InterMine of choice, you can run a query as normal - this could be a search, a list results page, a template, or a query in the query builder. Eventually you’ll be presented with an InterMine results table.
+
+> ### {% icon hands_on %} Hands-on: Run a query on Intermine
+> 4. Click on **Export** (top right). This will bring up a modal window.
+> 5. Select **Send to Galaxy** and double-check the *"Galaxy Location"* is correct.
+> 6. Click on the **Send to Galaxy** button on the bottom right of the pop-up window.
+>
+>    > ### {% icon tip %} Enable popups
+>    >
+>    > If you get an error when you click on the **Send to Galaxy** button, please make sure to allow popups and try again.
+>    {: .tip}
+>
+{: .hands_on}
+You have now exported your query results from InterMine to Galaxy.
+
+
+# Export identifiers into InterMine
+
+## Get data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Import some fly data from [Zenodo](https://zenodo.org/record/3407174) or from the data librar<>
+>
+>    ```
+>    https://zenodo.org/record/3407174/files/GenesLocatedOnChromosome4.tsv
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+> 2. Rename the dataset to `GenesLocatedOnChromosome4`
+>
+>    {% include snippets/rename_dataset.md %}
+>
+> 3. Inspect the data
+>
+{: .hands_on}
+
+The dataset contains the secondary identifier and the symbol of the *Drosophila melanogaster* genes and their location on the chromosome 4
+
+> ### {% icon question %} Questions
+>
+> Do the data contain the type, e.g `Protein` or `Gene`?
+>
+> > ### {% icon solution %} Solution
+> >
+> > No, they don't. So we have to specify it, when we create the InterMine Interchange file
+> >
+> {: .solution}
+>
+{: .question}
+
+## Create InterMine Interchange dataset
+
+We will use **Create InterMine Interchange Dataset** {% icon tool %} in order to generate an intermediate file which will be used to send the identifiers (e.g. gene identifiers) to InterMine. This file requires the identifier's type (e.g. `Gene`), the identifier (e.g `WBGene00007063`) and, optionally, the organims's name.
+
+> ### {% icon hands_on %} Hands-on: Generate InterMine file
+>
+> 1. {% tool [Create InterMine Interchange dateset](toolshed.g2.bx.psu.edu/repos/iuc/intermine_galaxy_exchange/galaxy_intermine_exchange/0.0.1) %} with the following parameters:
+>    - {% icon param-file %} *"Tabular file"*: select the `GenesLocatedOnChromosome4` dataset which contains some fly's genes
+>    - *"Feature Type Column"*: select a column from the input file which contains the identifer's type you are exporting to InterMine.
```

Comment:
Hi @danielabutano I could figure out what column should be selected here?

It's a very generic instruction, and in the GTN we usually prefer very specific ones (select col 1). Which column should be selected here?

Reply:
Hi @hexylena. thanks for reviewing this PR.
In this case you don't have to select any column and (as specified in the following sentence)

> because the `GenesLocatedOnChromosome4` dataset does not containt the type we have to edit it, in the *"Feature Type"*

Reply:
Ah ok, this was not clear for me from my reading. Is there any chance you could make a screenshot of the interface? That would help me ensure there are no misunderstandings for the other parameters (if i misunderstood this one, maybe i misunderstand the other descriptions) 

Then I can then write it in the GTN's preferred format/style.

Reply:
![image](https://user-images.githubusercontent.com/1678179/101629474-52a3ec00-3a19-11eb-984a-f21df15ff5ca.png)

Reply:
thanks!

Reply:
I try to explain in a different way. 
We need to provide Feature Type and Feature Identifier (Organism is not mandatory).
These values can be extraxted from the dataset provided (using the column selected in Feature Type column or Feature identifier column) or manually edited (in the Feature Type or Feature Identifier fields).
If you edit the value (in the Feature Type or Feature Identifier fields), the column selected is irrilevant.

Reply:
That makes sense. Does it function if both fields are specified (e.g. feature identifier column + feature identifier?) or what's the behaviour in that case?

Reply:
If both fields are specified, the field edited wins (as Feature Type column and Feature type in the screenshot).
This behavior is a bit weird, I know, it would be much much more clear, if the user could select "Nothing selected" in the column and edit "Gene" in the field

Reply:
Maybe we could fix that in the tool, provide the user a conditional like https://github.com/galaxyproject/tools-iuc/blob/1f35303af979c16d9a3126dbc882a59f686ace5d/tools/tbl2gff3/tbl2gff3.xml#L48 that one? This way we can ensure no unintentional/bad states.

Reply:
I agree with you that something like that might be more clear for the user